packages/experimental-app-router/src/server/routeHandler/index.ts

```diff
@@ -0,0 +1,25 @@
+import { notFound } from 'next/navigation.js';
+import { tokenHandler } from './tokenHandler.js';
+
+export async function GetFn(req: Request) {
+  const { pathname } = new URL(req.url);
+
+  switch (pathname) {
+    case '/api/faust/token': {
+      return tokenHandler(req);
+    }
+    default: {
+      return notFound();
+    }
+  }
+}
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export async function PostFn(req: Request) {
+  // Intentionally left empty if/until we have POST endpoints.
+}
+
+export const faustRouteHandler = {
```

Comment:
For more info on route handlers:

https://nextjs.org/docs/app/building-your-application/routing/route-handlers